ctdcal/ctd_plots.py

```diff
@@ -221,6 +228,185 @@ def _intermediate_residual_plot(
     return _save_fig(ax, f_out)
 
 
+def haversine(lat1, lon1, lat2, lon2, units="km"):
```

Comment:
Any reason `gsw.distance` is not suitable?

Reply:
It is!

Will be reworking and expanding the plotting capabilities for future release.